src/miscellaneous.jl

```diff
@@ -0,0 +1,13 @@
+# File containing code which I am currently unsure which file in which it belongs. Long-term functions here should be moved elsewhere.
+
+
+# If u0s are not given while conservation laws are present, throws an error.
+# If the system is nested, the function is skipped (conservation laws cannot be computed for nested systems).
+# Used in HomotopyContinuation and BifurcationKit extensions.
+function conservationlaw_errorcheck(rs, pre_varmap)
```

Comment:
Maybe just put this in the network api file near the conservation law code? Seems more appropriate to go with that code. Then we can drop having a misc file.

Reply:
happy to move it there.

(stability stuff is currently in the misc file as well, but yes, I agree that the misc file should be dropped, but figured if I put stuff somewhere it was likely you'd want it moved anyway)

Reply:
moved